kpi/views/tmp_browsers_kpi_2023.view.lkml

```diff
@@ -0,0 +1,100 @@
+view: tmp_browser_kpis_2023 {
+  derived_table: {
+    sql:
+      WITH
+      desktop AS (
+      SELECT submission_date,
+        'desktop' AS product,
+        COUNTIF(
+          active_hours_sum > 0 AND
+          COALESCE(
+            scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+            scalar_parent_browser_engagement_total_uri_count_sum,
+            0
+          ) > 0
+        ) AS dau,
+        AVG(COUNTIF(
+          active_hours_sum > 0 AND
+          COALESCE(
+            scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+            scalar_parent_browser_engagement_total_uri_count_sum,
+            0
+          ) > 0
+        )) OVER (ORDER BY submission_date ROWS BETWEEN 27 PRECEDING AND CURRENT ROW) AS dau_28ma
+      FROM `mozdata.telemetry.clients_daily`
+      WHERE submission_date >= DATE_SUB('2022-12-16', INTERVAL 27 DAY)
+      GROUP BY 1
+      ),
+
+      mobile AS (
+      SELECT submission_date,
+        'mobile' AS product,
+        COUNTIF(
+          `mozfun`.bits28.active_in_range(days_seen_bits, 0, 1) AND
```

Comment:
I am not familiar with the logic but `days_since_seen` column is also available in the `unified_metrics` table:  https://github.com/mozilla/bigquery-etl/blob/b4345961fde4b119dc6a1c9f361e0ddd9211fcb9/sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/query.sql#L65

Reply:
+1  

`mozfun.bits28.active_in_range(days_seen_bits, 0, 1)` [should be strictly equivalent](https://docs.telemetry.mozilla.org/cookbooks/clients_last_seen_bits.html#bits28active_in_range) to `days_since_seen = 0` however the latter might be more transparent.

Reply:
That's a good idea, y'all. For the sake of this temporary dashboard though, I'd like to stick with this definition because it's the one that's used in the 2023 [Mobile KPI Analysis](https://docs.google.com/document/d/1aNazhOEPexxiG_gAxIFJK-sHIHhEm0clXtvK1TtdAWw/edit#) -- the query is defined on [page 4 of this doc](https://drive.google.com/drive/folders/1WBRFAhiULyym9L08CWdFlCnhJmqnd_EB). We can keep our paper trail slightly simpler by sticking with this version until we update to use the new tables.